testdata/baselines/reference/submodule/conformance/asyncConstructor_es6.js.diff

```diff
@@ -0,0 +1,10 @@
+--- old.asyncConstructor_es6.js
++++ new.asyncConstructor_es6.js
+@@= skipped -7, +7 lines =@@
+ 
+ //// [asyncConstructor_es6.js]
+ class C {
+-    constructor() {
++    async constructor() {
```

Comment:
This seems like a bug or something; the constructor is getting the modifiers from another node. The same happens below for `accessor constructor()` too.

Reply:
Not a bug, per se, just different from Strada. We already report an error that `async` isn't allowed on `constructor`, so this change seemed to be acceptable.

Reply:
Is it a problem for our emitter to technically output invalid code?

Reply:
No, we do it all the time in the presence of errors. We make no guarantees on whether syntactically invalid input code will produce syntactically valid output code.